contrib/sirupsen/logrus/logrus.go

```diff
@@ -34,7 +49,11 @@ func (d *DDContextLogHook) Fire(e *logrus.Entry) error {
 	if !found {
 		return nil
 	}
-	e.Data[ext.LogKeyTraceID] = span.Context().TraceID()
-	e.Data[ext.LogKeySpanID] = span.Context().SpanID()
+	if cfg.log128bits && span.Context().TraceID() != tracer.TraceIDZero {
```

Comment:
Why do we check if trace ID is zero here, but not in `log/slog`?

Reply:
We do!
I actually introduced the ! TraceIDZero check here because I was inspired by the check in slog.
See line 74 of slog.go file.